classes/PodsInit.php

```diff
@@ -293,8 +293,8 @@ public function register_assets() {
 			wp_register_style( 'jquery-ui-timepicker', PODS_URL . 'ui/js/timepicker/jquery-ui-timepicker-addon.min.css', array(), '1.6.3' );
 		}
 
-		wp_register_script( 'pods-select2', PODS_URL . 'ui/js/select2/select2.min.js', array( 'jquery', 'pods-i18n' ), '4.0.3' );
-		wp_register_style( 'pods-select2', PODS_URL . 'ui/js/select2/select2.min.css', array(), '4.0.3' );
+		wp_register_script( 'pods-select2', PODS_URL . 'ui/js/selectWoo/selectWoo.min.js', array( 'jquery', 'pods-i18n' ), '4.0.3' );
+		wp_register_style( 'pods-select2', PODS_URL . 'ui/js/selectWoo/selectWoo.min.css', array(), '4.0.3' );
```

Comment:
I know it wasn't there before, but this might be a good time to add a SCRIPT_DEBUG check to not use the minified versions